frontend/src/app/modules/home/modals/dashboard/update-dashboard.component.ts

```diff
@@ -32,8 +32,9 @@ export class UpdateDashboardComponent implements OnInit {
                 this.dashboard.name,
                 [
                     Validators.required,
-                    Validators.minLength(5),
-                    Validators.maxLength(50)
+                    Validators.minLength(3),
+                    Validators.maxLength(50),
+                    Validators.pattern('^[a-zA-Z0-9_. ]*$')
```

Comment:
Move the regex to meaningful constant and import it in all places where you need it